text/setup.py

```diff
@@ -22,9 +22,24 @@
     'scipy',
     'pandas',
     'scikit-learn',
-
+    'fairscale',
+    'scikit-image',
+    'boto3',
+    'pytest',
```

Comment:
We should not depend on pytest + it is present multiple times in this list.

Reply:
Pytest removed.